android/assets/jsons/translations/Lithuanian.properties

```diff
@@ -2,2124 +2,1400 @@
+You have captured our cities! = Jūs užgrobėte mūsų miestus!
+We applaud your liberation of our conquered cities! = Mes dėkingi jums už išlaisvintus mūsų miestus!
+We applaud your liberation of conquered cities! =  Mes dėkingi jums už išlaisvintus miestus!
+Years of peace have strengthened our relations. = Ilgi metai taikos sutvirtino mūsų santykius
+Our mutual military struggle brings us closer together. = Mūsų bendri kariniai veiksmai vienija
+We have signed a public declaration of friendship = Mes viešai skelbiame draugystę
+You have declared friendship with our enemies! = Jūs draugaujate su mūsų priešais!
+You have declared friendship with our allies = Jūs draugaujate su mūsų sąjungininkais
+Our open borders have brought us closer together. = Atviros mūsų sienos mus suartino
+Your so-called 'friendship' is worth nothing. = Jūsų apsimestinis draugiškumas bevertis
+You have publicly denounced us! = Jūs mus viešai pasmerkėte!
+You have denounced our allies = Jūs pasmerkėte mūsųs sąjungininkus
+You have denounced our enemies = Jūs pasmerkėte mūsų priešus
+You betrayed your promise to not settle cities near us = Jūs sulaužėte pažadą nesikurti šalia mūsų
+You fulfilled your promise to stop settling cities near us! = Jūs įvykdėte pažadą nesikurti šalia mūsų
+You refused to stop settling cities near us = Jūs atisakėte nesikurti šalia mūsų
+Your arrogant demands are in bad taste = Jūsų pasipūtėliški reikalavimai nepriimtini
+Your use of nuclear weapons is disgusting! = Jūsų naudojimasis branduoliniais ginklais yra šlykštus!
+You have stolen our lands! = Jūs užgrobėte mūsų žemes!
+You gave us units! = Jūs davėte mums dalinius!
+You destroyed City-States that were under our protection! = Jūs sugriovėte mūsų ginamus laisvuosius miestus!
+You attacked City-States that were under our protection! = Jūs atakavote mūsų ginamus laisvuosius miestus!
+You demanded tribute from City-States that were under our protection! = Jūs reikalavote duoklės iš mūsų ginamų laisvųjų miestų!
+You sided with a City-State over us = Jūs susitarėte su laisvuoju miestu apeidami mus
+You returned captured units to us = Jūs grąžinote musų dalinius iš nelaisvės
+
+Demands = Reikalavimai
+Please don't settle new cities near us. = Nesikurkite šalia mūsų.
+Very well, we shall look for new lands to settle. = Gerai, mes ieškosime kitų žemių gyvenvietėms.
+We shall do as we please. = Mes elgsimės kaip panorėsim.
+We noticed your new city near our borders, despite your promise. This will have....implications. = Mes aptikome jūsų miestą prie mūsų sienų. Tai neliks... be pasekmių.
+I've been informed that my armies have taken tribute from [civName], a city-state under your protection.\nI assure you, this was quite unintentional, and I hope that this does not serve to drive us apart. = Man pranešė, kad mano kariuomenė ėmė duoklę iš jūsų ginamo laisvojo miesto.\nPatikinu jus, tai buvo nesusipratimas ir tikiuosi kad tai nebus priešastimi mūsų santykių pablogėjimui.
```

Comment:
No need to be sorry. This would be the first one, where "[civName]" just needs to appear on the right as well

Reply:
fixed it